compiler/noirc_evaluator/src/acir/mod.rs

```diff
@@ -411,14 +413,14 @@ impl<'a> Context<'a> {
         let mut warnings = Vec::new();
         // Disable the side effects if the binary instruction does not require them
         let one = self.acir_context.add_constant(FieldElement::one());
```

Comment:
We could also inline this `one` where it's called, to avoid creating a "one" constant for every instruction in the program.